### PR TITLE
bugfix/dtl-user-testing

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/decentlab/IAM/uplink.js [hunk 1]
```diff
@@ -216,7 +216,11 @@ function consume(event) {
   data.rawPir = sample.raw_ir_reading;
 
   // Lifecycle values
-  lifecycle.voltage = sample.battery_voltage;
+  lifecycle.voltage = Math.round(sample.battery_voltage * 100) / 100;
+  lifecycle.batteryLevel = Math.round(
+    ((3 - lifecycle.voltage) * 100 - 100) * -1,
```
Comment (mariusgiger) [inline types/decentlab/IAM/uplink.js:221]: explain the calculation in a comment

File: types/talkpool/oy1700/addon.schema.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "particleCount5": {
+      "title": "Particulate matter count 5",
+      "unit": "μm",
+      "type": "integer",
+      "description": "Particles less than 5 μm in diameter",
+      "minimum": 0,
+      "maximum": 10000
+    },
+    "particleCount5Bigger": {
+      "title": "Particulate matter count 5 <",
```
Comment (mariusgiger) [inline types/talkpool/oy1700/addon.schema.json:48]: more than would be >5

File: types/talkpool/oy1700/uplink.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ function consume(event) {
   const { port } = event.data;
   const bits = Bits.hexToBits(payload);
   const data = {};
+  const addon = {};
```
Comment (mariusgiger) [inline types/talkpool/oy1700/uplink.js:6]: what is addon?

**Review by mariusgiger — COMMENTED** [review]
looks good, a few comments

File: types/talkpool/oy1700/uplink.js (hunk 3 above)
Comment (J2thatsme) [reply types/talkpool/oy1700/uplink.js:6]: The sensor does usually not send this message. So i put it as add-on better names are welcomed ^^

Comment (mariusgiger) [reply types/talkpool/oy1700/uplink.js:6]: ask yourself what this non standard message usually contains and name the variable accordingly (e.g. reportData, calibrationData etc.)

**Review by mariusgiger — DISMISSED** [review]

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**